packages/signature/src/lib/utils.ts

```diff
@@ -344,3 +345,17 @@ export const generateJwk = async <T = Jwk>(
 export const nowSeconds = (): number => Math.floor(Date.now() / 1000)
 
 export const getPublicKey = (key: Jwk): PublicKey => publicKeySchema.parse(key)
+
+export const requestWithoutWildcardFields = (
+  request: object,
+  wildcardPaths: string[] = [],
+  allowedPaths: string[] = []
+): object => {
+  const validPaths = wildcardPaths.filter((path) => allowedPaths.includes(path))
+  const requestCopy = JSON.parse(JSON.stringify(request))
```

Comment:
lol, you know JSON parse and stringify is extremely slow and error prone, right?

Just use lodash.
```suggestion
  const requestCopy = cloneDeep(request)
```

Reply:
No, I did not. I used the pattern that [was already there](https://github.com/narval-xyz/armory/blob/d40d992ca730fef22d7b1cbb78779f6cc7bd8957/packages/signature/src/lib/verify.ts#L330-L333) in verify function. Ty for the insight